eatngo-core/src/main/kotlin/com/eatngo/product/event/ProductEventListener.kt

```
@@ -38,12 +41,44 @@ class ProductEventListener(
     }
 
     private fun processStockDecrease(orderItem: OrderItem) {
-        inventoryCachePersistence.decreaseStock(orderItem.productId, orderItem.quantity)
+        try {
+            inventoryCachePersistence.decreaseStock(orderItem.productId, orderItem.quantity)
+        } catch (e: StockNotFound) {
```

detekt (1.23.8)

[warning] The caught exception is swallowed. The original exception could be lost.

(detekt.exceptions.SwallowedException)